oasislmf/preparation/gul_inputs.py

```diff
@@ -54,7 +43,7 @@
 ]
 
 
-def process_group_id_cols(group_id_cols, exposure_df_columns, has_correlation_groups):
+def process_group_id_cols(group_id_cols, exposure_df_columns, has_correlation_groups, peril_correlation_group_col='peril_correlation_group'):
```

Comment:
I don't see if this function is called with a different peril_correlation_group_col.
If not, and if we don't have a use case in mind where this is useful I would use a global variable to name the column.

Reply:
thanks, yes, a `peril_correlation_group_col` different from the default value is never used and I don't see a use for that flexibility in the short term.
peril correlation groups are defined uniquely and we currently have no plans to implement multiple peril correlation group layers.
so, I'll move that value to a global variable as suggested.